client/listeners/index.js

```diff
@@ -55,7 +55,7 @@ const {Events} = DOM;
 
 const EventsFiles = {
     mousedown: exec.with(execIfNotUL, setCurrentFileByEvent),
-    click: execAll([onClick, unselect]),
+    click: execAll([onClick, function(e)  { if (window.innerWidth > CloudCmd.MOBILE_ONE_PANEL_WIDTH) { unselect(e); } }]),
```

Comment:
```suggestion
    click: execAll([onClick, exec.with(execIfNotMobile, unselect)]),
```

Reply:
And inside `execIfNotMobile` condition that reads css variable. In this case we will have one place of truth that knows if we in mobile mode: and this is CSS.

Reply:
I looked/hoped there is already a flag/check for mobile UI.

I understand "execIfNotMobile" is a suggested one and not existing one.

It is a bit tied to the previous comment.

I can try to create it with the existing code logic or with a new one described on previous comment.